src/main/webapp/user-page.jsp

```diff
@@ -92,7 +92,7 @@ String uploadUrl = blobstoreService.createUploadUrl("/my-form-handler"); %>
 
   <!-- <form id="message-form" action="/messages" method="POST" class="hidden"> -->
 
-    <form id="message-form" action="<%= uploadUrl %>" method="POST" enctype="multipart/form-data">
+    <form id="message-form" action="<%= uploadUrl %>" method="POST" enctype="multipart/form-data" class="hidden">
```

Comment:
what does class="hidden" do again?

Reply:
it's for when the user page to see if it's your page; if it is your page, it'll show the input / message boxes, but if it isn't, it won't show them :)

Reply:
ohhhh nice! good call 👍